lib/composer

```diff
@@ -64,9 +64,14 @@ function install_composer_deps() {
         status "Bundling additional extensions"
         for ext in $required_extensions; do
             local apt_deps=""
+            local is_embedded=""
             local ext_version=$(jq --raw-output ".require | .[\"ext-${ext}\"]" < "${BUILD_DIR}/composer.json")
 
-            if [ "$(is_embedded_extension "${ext}")" = "true" ] ; then
+            is_embedded="$(is_embedded_extension ${ext})"
+            rc=$?
+            [ ${rc} -ne 0 ] && error "error while trying to identify if ${ext} is embeded in runtime."
```

Comment:
Small typo here: `embeded` -> `embedded`.

Reply:
Oh thanks a lot! I'll correct this spelling mistake.